hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
```

Comment:
Since we're fixing a bug rather than adding a feature, this should be a `patch` release.